src/nano_id_cc/core.cljs

```diff
@@ -49,17 +50,23 @@
     (str (format-time time))))
 
 
+(defn escape
+  [s]
+  (st/escape s {\' "\\'"
+                \" "\\\""}))
+
+
 (defn code-sample
   [{:keys [alphabet length]}]
   (let [custom? (not= alphabet defaults/alphabet)
         len     (when (not= length defaults/length) length)
         nano-id (nano-id.custom/generate alphabet)
-        id      (nano-id length)]
+        id      (escape (nano-id length))]
```

Comment:
Because `id` represents a string as it is, we shouldn't care about escaping here. Moreover it's just a comment.